Author: Martin Pitt <mpitt@debian.org>
Description: Update pg_service.conf example to tell the Debian specific file location.
Forwarded: No, Debian specific

--- a/src/interfaces/libpq/pg_service.conf.sample
+++ b/src/interfaces/libpq/pg_service.conf.sample
@@ -7,8 +7,8 @@
 # the pattern  "param=value".  A sample configuration for postgres is 
 # included in this file.  Lines beginning with '#' are comments.
 #
-# Copy this to your sysconf directory (typically /usr/local/pgsql/etc) and
-# rename it pg_service.conf.
+# Copy this to /etc/postgresql-common/ (or select its location with the
+# PGSYSCONFDIR environment variable) and rename it pg_service.conf.
 #
 #
 #[postgres]
